init.lua

```diff
@@ -517,7 +518,7 @@ function InitSocket()
 	local url = "ws://" .. host .. ":" .. port
 	Log.Info("Connecting to " .. url .. "...")
 
-	sock = pollnet.open_ws(url)
+	sock = pollnet.open_ws(url, 1048576)
```

Comment:
```suggestion
	sock = pollnet.open_ws(url, 10 * 1024 * 1024)
```

Easier to tell that it's a 10MB buffer with this.